internal/utils/docker.go

```diff
@@ -244,7 +245,7 @@ func DockerPullImageIfNotCached(ctx context.Context, imageName string) error {
 	return DockerImagePullWithRetry(ctx, imageUrl, 2)
 }
 
-func DockerStart(ctx context.Context, config container.Config, hostConfig container.HostConfig, containerName string) (string, error) {
+func DockerStart(ctx context.Context, config container.Config, hostConfig container.HostConfig, containerName string, aliases []string) (string, error) {
```

Comment:
```suggestion
func DockerStart(ctx context.Context, config container.Config, hostConfig container.HostConfig, networkingConfig *network.NetworkingConfig) (string, error) {
```

Would be nice to eventually arrive at this signature